pkg/controller/reconciler/deployment_reconciler_test.go

```diff
@@ -145,14 +152,14 @@ func Test_DeploymentReconciler_ChangeEnvironmentAndApplication(t *testing.T) {
 
 	// Deployment will be waiting for container to complete deployment.
 	annotations := waitForStateUpdating(t, client, name)
-	require.Equal(t, "/planes/radius/local/resourceGroups/default-deployment-change-envapp", annotations.Status.Scope)
+	require.Equal(t, "/planes/radius/local/resourcegroups/default-deployment-change-envapp", annotations.Status.Scope)
 	require.Equal(t, "/planes/radius/local/resourceGroups/default/providers/Applications.Core/environments/default", annotations.Status.Environment)
-	require.Equal(t, "/planes/radius/local/resourceGroups/default-deployment-change-envapp/providers/Applications.Core/applications/deployment-change-envapp", annotations.Status.Application)
+	require.Equal(t, "/planes/radius/local/resourcegroups/default-deployment-change-envapp/providers/Applications.Core/applications/deployment-change-envapp", annotations.Status.Application)
```

Comment:
Should we make everything lower case?
```suggestion
	require.Equal(t, "/planes/radius/local/resourcegroups/default-deployment-change-envapp/providers/applications.core/applications/deployment-change-envapp", annotations.Status.Application)
```

Reply:
Just saw that you lower-cased the **resourceGroups**... You can ignore the comment above.